### PR TITLE
assignment 14 changes

### PR DESCRIPTION
**Pull request by harshalkotkar**

### CODE REVIEW

File: HARSHAL/javascript/assignment14/assignment14Q1.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<html>
```
Comment (poonam-khode) [inline HARSHAL/javascript/assignment14/assignment14Q1.html]: add <!Doctype html>

File: HARSHAL/javascript/assignment14/assignment14Q1.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    <head>
+        <title>Inheritence</title>
```
Comment (poonam-khode) [inline HARSHAL/javascript/assignment14/assignment14Q1.html:3]: change title(give title according to the program)

**Review by MKadeer17 — APPROVED** [review]

File: HARSHAL/javascript/assignment14/assignment14Q1.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                    document.write("<br>Roll No " + this.RollNo);
+                    document.write("<br>Name " + this.Name);
+                }
+            }
+            //Marks is a derived class/child class
+            class Marks extends student {
+                //Construcor Overrriding
+                constructor(RollNo, Name, Marks) {
+                    super(RollNo, Name); //as we are inhereting properties of parent class we need to use super method
+                    // this.RollNo = RollNo;
```
Comment (poonam-khode) [inline HARSHAL/javascript/assignment14/assignment14Q1.html:23]: remove these comments

**Review by poonam-khode — CHANGES_REQUESTED** [review]

File: HARSHAL/javascript/assignment14/assignment14Q1.html (hunk 3 above)
Comment (harshalkotkar) [reply HARSHAL/javascript/assignment14/assignment14Q1.html:23]: why?

Comment (harshalkotkar) [reply HARSHAL/javascript/assignment14/assignment14Q1.html:23]: its recommended, as the question was asked to do necessary changes so its valid to comment ?

Comment (poonam-khode) [reply HARSHAL/javascript/assignment14/assignment14Q1.html:23]: there is no need to push commented code

Comment (harshalkotkar) [reply HARSHAL/javascript/assignment14/assignment14Q1.html:23]: but why ?

Comment (Snehal-113) [reply HARSHAL/javascript/assignment14/assignment14Q1.html:23]: poonam proper comments are expected with any code

Comment (poonam-khode) [reply HARSHAL/javascript/assignment14/assignment14Q1.html:23]: ok mam

**Review by Snehal-113 — APPROVED** [review]

**Review by poonam-khode — APPROVED** [review]

**Pull request merged**